pkg/cmdlets/auth/signup.go

```diff
@@ -34,7 +28,8 @@ type signupInput struct {
 
 // Signup will prompt the user to create an account
 func Signup(cfg keypairs.Configurable, out output.Outputer, prompt prompt.Prompter, auth *authentication.Auth) error {
-	accepted, err := promptTOS(cfg.ConfigPath(), out, prompt)
+	tos := prompts.NewOnlineTOS()
+	accepted, err := prompts.PromptTOS(tos, out, prompt)
```

Comment:
As noted on the other use of `PromptTOS()`, this doesn't seem like it needs to live here. This logic can live in the prompts package with a convenient function that consumers like this would call without the need to supply the tos text.